manual/src/html_processing/src/process_manual.ml

```diff
@@ -19,9 +19,19 @@ let index_title = "Home"
 let archives =
   ["refman-html.tar.gz"; "refman.txt"; "refman.pdf"; "refman.info.tar.gz"]
 
+(* WARNING these are sensitive to Hevea fluctuations: *)
+(* "long" space is either " " (hevea 2.32) or "\u{2003}" (hevea 2.35) *)
+let preg_emspace = "\\(\u{2003}\\| \\)"
+(* What hevea inserts between "Chapter" and the chapter number: *)
+let preg_chapter_space = "\\( \\|\u{2004}\u{200d}\\)"
+let writtenby_css = "span.c010" (* "span.c009" for hevea 2.32 *)
+
+let preg_anyspace = "\\( \\|\u{2003}\\| \\|\u{2004}\\)+"
```

Comment:
I think we could add \u{00A0} (non-breaking space) and \u{202F} (narrow (outside of Mongolian) non-breaking space) to be future proof with the fixed version of hevea.

Reply:
ok. Note that \u{00A0} is already present in the regexp, I entered in directly as a UTF char; but github displays it as a normal space. In this direction, there are actually other "space" chars:
```
0x1680; (* " " = OGHAM SPACE MARK *)
0x2000; (* " " = EN QUAD *)
0x2001; (* " " = EM QUAD *)
0x2002; (* " " = EN SPACE *)
0x2003; (* " " = EM SPACE *)
0x2004; (* " " = THREE-PER-EM SPACE *)
0x2005; (* " " = FOUR-PER-EM SPACE *)
0x2006; (* " " = SIX-PER-EM SPACE *)
0x2007; (* " " = FIGURE SPACE *)
0x2008; (* " " = PUNCTUATION SPACE *)
0x2009; (* " " = THIN SPACE *)
0x200a; (* " " = HAIR SPACE *)
0x2028; (* " " = LINE SEPARATOR *)
0x2029; (* " " = PARAGRAPH SEPARATOR *)
0x202f; (* " " = NARROW NO-BREAK SPACE *)
0x205f; (* " " = MEDIUM MATHEMATICAL SPACE *)
0x3000; (* "　" = IDEOGRAPHIC SPACE *)
```

Reply:
In principle, I would be fine with including 0x2000 to 0x200A and 0x202f. The semantic of the other white space characters make them quite unlikely to ever be used as spaces between English words.

Reply:
ok, I just did this